core/tests/test_utils.py

```diff
@@ -91,6 +91,18 @@
 # We are using the b' prefix as all the stdouts are in bytes.
 LOG_LINE_PREFIX = b'LOG_INFO_TEST: '
 
+# List of model classes that don't have Takeout related class methods
+# defined because they're not used directly but only as a base classes for
+# the other models.
+BASE_MODEL_CLASSES_WITHOUT_TAKEOUT_POLICIES = (
```

Comment:
This constant is also used by Wipeout so I would maybe use `BASE_MODEL_CLASSES_WITHOUT_DATA_POLICIES` instead to make it clear that this is not only Takeout constant.

Reply:
Done! Let me know if this looks good!

Reply:
Looks good I have left one last comment about the comment above this.